sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -569,7 +596,8 @@ def share(self, name, version, *, share_with_name, share_with_version, registry_
 
         #  Get workspace info to get workspace GUID
         workspace = self._service_client.workspaces.get(
-            resource_group_name=self._resource_group_name, workspace_name=self._workspace_name
+            resource_group_name=self._resource_group_name,
+            workspace_name=self._workspace_name,
         )
         workspace_guid = workspace.workspace_id
```

Comment:
Would this potentially cause an issue? Workspace doesnt have a "workspace_id" member so should this be "_workspace_id" or should workspace class include "workspace_id" in its __init__ method

Reply:
Let's revisit it, this is not directly related to the scope of this PR, but happy to chat, also it seems per your investigation the share feature for model works....